packages/tm-grammars/index.js

```diff
@@ -3913,6 +4015,9 @@ export const injections = [
     embeddedIn: [
       'ts-tags',
     ],
+    funding: {
+      ko_fi: 26,
```

Comment:
These numbers are due to an error in the upstream YAML file. I submitted a PR to fix: https://github.com/0x00000001A/es6-string-html/pull/66